File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L1/01/L1.ts
import { Console } from 'console';

const console = new Console(process.stdout, process.stderr);

console.log("Miles\t\tKilometers");
console.log("-------------------------------");

let miles = 1;

while (miles <= 10) {
    console.log(`${miles}\t\t${miles * 1.609}`);
    miles++;
}

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L1/01/L1.kt
package IR

object L1 {
    @JvmStatic
    fun main(args: Array<String>) {
        println("Miles\t\tKilometers")

        println("-------------------------------")

        var miles = 1

        while (miles <= 10) {
            println(miles.toString() + "\t\t" + miles * 1.609)
            miles++
        }
    }
}
